Automattic/jetpack
# Remove the upgrade nudge for the Premium Block

Relates to https://github.com/Automattic/gold/issues/11
Fixes https://github.com/Automattic/jetpack/issues/30349

## Proposed changes:

Removes the upgrade nudge for the Premium block

| Before  | After |
| ------------- | ------------- |
| <img width="1372" alt="Screenshot 2023-05-01 at 4 44 22 PM" src="https://user-images.githubusercontent.com/7076981/235537005-552bcc78-e773-4f20-84b7-f934e42f2610.png">  | <img width="1476" alt="Screenshot 2023-05-01 at 4 43 54 PM" src="https://user-images.githubusercontent.com/7076981/235537001-72649882-6edc-4e6d-8719-4ae62ee8d886.png">  | 

### Other information:

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Add the Premium block to a site with a free plan
* Verify that the upgrade nudge is no longer present
* Open the post in an incognito window
* Checkout to ensure the block works as expected
